TuneUp/TuneUpWindow.xaml

```diff
@@ -74,6 +75,14 @@
                     </Trigger>
                 </Style.Triggers>
             </Style>
+            <CollectionViewSource Source="{Binding ProfiledNodes}" x:Key="ProfiledNodesCollection2">
+                <CollectionViewSource.GroupDescriptions>
+                    <PropertyGroupDescription PropertyName="State" />
+                </CollectionViewSource.GroupDescriptions>
+                <CollectionViewSource.SortDescriptions>
+                    <componentmodel:SortDescription PropertyName="State" Direction="Descending"/>
```

Comment:
why is this set descending here? Then later cleared and set ascending?

Reply:
Whoops, I should have removed this. ProfiledNodesCollection**2** was just part of a quick test of how to implement group sort. It is removed now.